### PR TITLE
Tooltip for peers (Closes #91)

### LINKED ISSUE #91: Nicer lists and AS-tooltips for peers seen hijack Updates/Withdrawals and distinct values
**Is your feature request related to a problem? Please describe.**
In Hijack view, the lists of the peers seen hijack updates and withdrawals are not beautified, and the tooltip is missing.

**Describe the solution you'd like**
When someone presses on the button(s) to show the list(s), show the list as follows:
- AS1
- ...
- ASN
(make them scrollable, to fit in the page). Also:
- keep the lists sorted (ascending order)
- add tooltips for the ASNs involved.
Minor: also please change the name from "Seen Hijack BGP update" --> "Seen Hijack BGP Announcement".
Witthdrawals are also updates.

Follow the same principle for distinct value lists in BGP updates and hijacks.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by pgigis**
### Description of PR
Tooltip support for peers seen BGP Update/Withdrawal on hijack view.
Resolves issue https://github.com/FORTH-ICS-INSPIRE/artemis/issues/91.

What component(s) does this PR affect?

- [ ] Back-End (Database, Microservices, Containers, etc)
- [x] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #91

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vkotronis** [comment]
@pgigis Please check the rest needed for issue #91:
* please change the name from "Seen Hijack BGP update" --> "Seen Hijack BGP Announcement". Witthdrawals are also updates.
* Follow the same principle for distinct value lists in BGP updates and hijacks, to beautify them as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## [UNRELEASED] (NAME) - YYYY-MM-DD
 ### Added
 - TBD (Added a new feature)
+- Tooltip support for peers seen BGP Update/Withdrawal on hijack view.
```
Comment (vkotronis) [inline CHANGELOG.md]: BGP Announcement/Withdrawal

File: frontend/webapp/views/main/templates/hijack.htm [hunk 2]
```diff
@@ -677,8 +673,32 @@ <h1 class="page-header">Viewing Hijack <small id="hijack_status"></small></h1>
...
+                var peers_seen = data['peers_seen'].sort(function(a,b){ return a - b;});
+                var css_start = ['<div class="row">', '</div>'];
+                var peers_html = "";
+                for (var i = 0; i < peers_seen.length; i++) {
+                    if(peers_html == ""){
+                        peers_html += css_start[0];
+                    }else if(i % 4 == 0){
+                        peers_html += css_start[1] + "</br>" + css_start[0];
+                    }
+                    peers_html += '<div class="col-lg-3"><cc_as><input asn="true" class="form-control" style="text-align:center;" type="text" value="'+ peers_seen[i] +'" readonly></div></cc_as>';
```
Comment (vkotronis) [inline frontend/webapp/views/main/templates/hijack.htm]: do the same for the distinct values lists in BGP updates and hijacks tables (under them)

Comment (slowr) [reply frontend/webapp/views/main/templates/hijack.htm]: also for string concatenation:
http://mattsnider.com/use-a-string-buffer-for-better-performance/

**Pull request merged**